README.md

```diff
@@ -207,6 +208,29 @@ To make a breadcrumb current based on the query parameters do:
 breadcrumb "Posts", posts_path(order: :desc), match: {order: :desc}
 ```
 
+#### 2.1.4 :http_verbs
+
+**Loaf** allows you to match on multiple HTTP verbs in order to make a breadcrumb current with the `:http_verbs` option.
+
+The `:http_verbs` key accepts `:all` or an array with following values:
+
+* `:get`
+* `:post`
+* `:put`
+* `:patch`
+* `:delete`
+* `:head`
+
+Its default value is `[:get]`
+
+For example:
+
+```ruby
+http_verbs: %w[get head]
+http_verbs: %w[post get head]
```

Comment:
Since request methods are described with symbols, it'd be more appropriate and probably desirable to use symbol arrays here:

```ruby
http_verbs: %i[get head]
http_vers: %i[post get head]
```